src/test/kotlin/com/terraformation/backend/tracking/scenario/SiteCreatedStep.kt

```diff
@@ -0,0 +1,115 @@
+package com.terraformation.backend.tracking.scenario
+
+import com.terraformation.backend.db.DatabaseBackedTest
+import com.terraformation.backend.db.tracking.MonitoringPlotId
+import com.terraformation.backend.db.tracking.StratumId
+import com.terraformation.backend.db.tracking.SubstratumId
+import com.terraformation.backend.db.tracking.tables.references.PLANTING_SITES
+import com.terraformation.backend.db.tracking.tables.references.STRATA
+import com.terraformation.backend.db.tracking.tables.references.SUBSTRATA
+import com.terraformation.backend.point
+import com.terraformation.backend.rectangle
+import com.terraformation.backend.rectanglePolygon
+import com.terraformation.backend.tracking.model.MONITORING_PLOT_SIZE
+import com.terraformation.backend.util.toMultiPolygon
+import java.time.Instant
+import org.locationtech.jts.geom.MultiPolygon
+
+class SiteCreatedStep(
+    val scenario: ObservationScenario,
+    val survivalRateIncludesTempPlots: Boolean,
+) : NodeWithChildren<SiteCreatedStep.Stratum>() {
+  var boundary: MultiPolygon = rectangle(0)
+
+  private val test: DatabaseBackedTest
+    get() = scenario.test
+
+  fun stratum(number: Int, init: Stratum.() -> Unit) = initChild(Stratum(number), init)
+
+  override fun prepare() {
+    scenario.plantingSiteId =
+        test.insertPlantingSite(
+            boundary = rectangle(1),
+            gridOrigin = point(0),
+            survivalRateIncludesTempPlots = survivalRateIncludesTempPlots,
+        )
+  }
+
+  override fun finish() {
+    children.forEach { boundary = boundary.union(it.boundary).toMultiPolygon() }
+
+    test.dslContext
+        .update(PLANTING_SITES)
+        .set(PLANTING_SITES.BOUNDARY, boundary)
+        .where(PLANTING_SITES.ID.eq(scenario.plantingSiteId))
```

Comment:
If any of the boundary stuff ends up causing the scenarios to be slow, could also have a flag in ObservationScenario or something that allows the test to skip some of the boundary setting?